content/rest-api/overview.md

```diff
@@ -157,7 +157,7 @@ Our recommendations on maximum usage are:
 - a maximum of 4 concurrent API calls per PIM Connection
 - a maximum of 10 concurrent API calls in total on a PIM instance
 
-Our PIM Cloud service has currently no rate limit based on API requests per unit of time or per IP. 
+Our PIM Cloud service has currently no rate limit based on API requests per unit of time or per IP, except for the API dedicated to [updating & creating several attribute options](https://api.akeneo.com/api-reference.html#patch_attributes__attribute_code__options).
```

Comment:
The rate limit applies to both update endpoints 

- https://api.akeneo.com/api-reference.html#patch_attributes__attribute_code__options__code_
- https://api.akeneo.com/api-reference.html#patch_attributes__attribute_code__options

 I believe the message needs to be modified. I will coordinate with Christophe on this